FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -265,27 +304,22 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <note>3 Miracles of Jesus, corresponding to the Nos. 3, 5 and 10 in <bibl><ptr target="bm:Grebaut1920MiraclesJesusII"/><citedRange unit="page">589-592</citedRange></bibl>, 
                             <bibl><ptr target="bm:Grebaut1920MiraclesJesusII"/><citedRange unit="page">605-608</citedRange></bibl> and
                             <bibl><ptr target="bm:Grebaut1920MiraclesJesusII"/><citedRange unit="page">646-649</citedRange></bibl>.</note>
+                    </msItem>
+                    </msContents>
+                        <physDesc>
+                            <objectDesc>
+                                <layoutDesc><layout columns="2" writtenLines="25"></layout></layoutDesc><!-- copied from part 2 - or different? -->
+                            </objectDesc>
                         <handDesc><handNote xml:id="h3" script="Ethiopic">
                             <ab type="script"/>
                             <locus from="122rb" to="122vb"/> and <locus from="129ra" to="131ra"></locus>
                             <desc>
                             </desc>
                         </handNote>    </handDesc>
```

Comment:
`handDesc `like `layout `all belong to `physDesc`, so `msItem `and `msContents `must be closed before `physDesc `starts